runtime/compiler/runtime/MetricsServer.hpp

```diff
@@ -230,6 +241,10 @@ class HttpGetRequest
    char _httpVersion[4]; // 1.0 1.1  2.0, etc
    size_t _msgLength; // How much of the buffer is filled
    char _buf[BUF_SZ]; // Buffer for holding incoming data
+   BIO *_ssl;
+
+   std::string _response;
```

Comment:
This is a string object that needs to be destroyed when `HttpGetRequest` is freed. Note that in the current version of the code we do not call any destructor. In the old code it was not needed because there were no embedded objects.